### PR TITLE
File Upload Component

### PR DESCRIPTION
**Pull request by sachasmart-weavik**
### Description

> File upload component without IPFS integrated

Figma: https://www.figma.com/file/g7Edk2JCOMLpRAw9v8NJ4c/FA-Product-Wireframe---working-file?node-id=136%3A6292
### Checklist
ezgif com-gif-maker (8)

- [ ] `CHANGELOG` has been updated (_if appropriate_)
- [ ] Environment variables updated (_if appropriate_)

### CODE REVIEW

File: packages/client3/components/design-system/file-upload/file-upload.tsx [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+    if (formContext !== null) {
+      _errorMessage = formContext.formState.errors[name]?.message;
+    }
+    const isError = !!_errorMessage;
+
+    return (
+      <>
+        <div className="flex w-full items-center justify-center">
+          <label
+            htmlFor="dropzone-file"
```
Comment (v1n33th) [inline packages/client3/components/design-system/file-upload/file-upload.tsx:28]: I think this should be `file` as this label is for the input field right?

Comment (v1n33th) [reply packages/client3/components/design-system/file-upload/file-upload.tsx:28]:
```suggestion
            htmlFor="file"
```

File: packages/client3/components/design-system/file-upload/file-upload.tsx [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+              "border border-dashed",
+              isError ? "border-state-error" : "border-dark-70",
+              disabled && "opacity-50"
+            )}
+          >
+            <div className="flex flex-col items-center justify-center pt-5 pb-6">
+              <Icon
+                name={"AddCircle"}
+                size="md"
+                className={clsx("my-[10px]")}
```
Comment (v1n33th) [inline packages/client3/components/design-system/file-upload/file-upload.tsx:40]: 10px is 2.5 in tailwind's mapping. So this could be replaced to my-2.5

File: packages/client3/components/design-system/file-upload/types/file-upload.types.ts [hunk 3]
```diff
@@ -0,0 +1,7 @@
+export interface FileUploadProps {
+  id: string;
+  label: string;
+  name: string;
+  onChange: (event: React.ChangeEvent<HTMLInputElement>) => void;
+  value: string;
+}
```
Comment (v1n33th) [inline packages/client3/components/design-system/file-upload/types/file-upload.types.ts:7]: Is this file needed as I think the fileupload props are already declared within file

**Review by v1n33th — APPROVED** [review]
Awesome..Just some minor questions

File: packages/client3/components/design-system/file-upload/file-upload.tsx (hunk 2 above)
Comment (sachasmart-weavik) [reply packages/client3/components/design-system/file-upload/file-upload.tsx:40]: Lol thanks :P, I've started just going with the pixels because its easier to reference from the figma.

File: packages/client3/components/design-system/file-upload/types/file-upload.types.ts (hunk 3 above)
Comment (sachasmart-weavik) [reply packages/client3/components/design-system/file-upload/types/file-upload.types.ts:7]: Perfect, added that just in case we need to doooo...something?
